### DISCUSSION TITLE
Methyl-benzoate bonded construction & mapping

### DISCUSSION THREAD

**Discussion by vaidyanathanms** [ricalessandri/Martini3-small-molecules#24] [Q&A] [answered]
In the parameter file for methyl-benzoate,

; id   type    resnr   residu  atom    cgnr    charge
   1    N4a      1      MBZOA   O1      1        0
   2   TC5       1      MBZOA   R2      2        0
   3   TC5       1      MBZOA   R3      3        0
   4   TC5       1      MBZOA   R4      4        0

[bonds]
; i j   funct   length  force.c.
  1 2     1      0.389   5000 ; cog
  1 3     1      0.389   5000 ; cog

Why is there a bond between N4a and two different TC5 atoms? 

Wouldn't the structure then look like

           N4a
           /.      \  
      TC5  -  TC5
           \.         /
              TC5

Instead of 

         N4a
           |  
      TC5  -  TC5
           \.         /
              TC5

**Comment by ricalessandri** [comment, marked as answer]
Hi and thanks for the question. Yep, the structure looks like your top one. I've uploaded some more sketches to clarify further: https://github.com/ricalessandri/Martini3-mappings/blob/main/methyl-benzoate.pdf

This is because the -COOMe group is fairly rigid and this construction is very numerically stable (see, e.g., minute 11:16 of [this talk](https://www.youtube.com/watch?v=PfgEx72KxVg)).

Feel free to follow up, I'd be happy to clarify the choice further.

**Comment by vaidyanathanms** [comment]
Hi,

Thank you very much for the quick response. It makes sense now :).

Thank you,

Vaidyanathan M. S

**Comment by Clare-xiong** [comment]
I'm a little confused about the three TC5 beads. The carbon atom immediately connected to the -COOMe group (call it "alpha carbon"). Why do the TC5 map 2 non-hydrogen atoms like this :(1*C_beta + 1/2*C_alpha + 1/2 * C_gamma)? Why is it not like a benzene(TC5 bead map the -CH=CH- group)? When I map the molecule to the CG model, how should I deal with the benzene? Should I make the TC5 bead map like methyl-benzoate, or the benzene?

**Comment by ricalessandri** [comment]
The mapping is chosen to preserve the symmetry of the molecule. You can see for yourself by trying to draw mappings such as `1*C_alpha + 1*C_beta` etc. and compare them to the [recommended mapping](https://github.com/ricalessandri/Martini3-mappings/blob/main/methyl-benzoate.pdf).  Note that in either case the TC5 beads would be representing 2 x C atoms, i.e., the amount of non-hydrogen atoms that they should represent. They are just placed in a more symmetric way. So, my recommendation would be to follow the `1*C_beta + 1/2C_alpha + 1/2 * C_gamma` mapping so as to get a more symmetric CG model. Let me know if this clears your doubts (or not)!

**Comment by Clare-xiong** [comment]
If I understand correctly, you mean that there can be more than one way to map while guaranteeing that TC5 beads will represent two carbon atoms. The mapping method will be chosen based on considerations of symmetry. Am I right in my understanding?

**Comment by ricalessandri** [comment]
Yes, exactly.
